docs/topics/auth/passwords.txt

```diff
@@ -173,10 +216,32 @@ default PBKDF2 algorithm:
             'django.contrib.auth.hashers.CryptPasswordHasher',
         ]
 
-
 That's it -- now your Django install will use more iterations when it
 stores passwords using PBKDF2.
 
+Argon2
+~~~~~~
+
+Argon2 has three attributes that can be customized:
+
+#. ``time_cost`` controls the number of iterations within the hash
+#. ``memory_cost`` controls the size of memory that must be used during the
+   computation of the hash
+#. ``parallelism`` controls how many CPUs the computation of the hash can be
+   parallelized on.
+
+The default values of these attributes are probably fine for you.  If you
```

Comment:
It is a good idea to review the defaults for each major release.  The analogue of the current rule would be to increase `memory_cost` with approximately 20% for each major release.  Maybe it would be a better idea to use actual hardware as a reference: adjust the parameters such that the hash takes a certain amount of milliseconds on the cheapest Amazon EC2 instance.